<launch>
	<param name="use_sim_time" value="true" />
	<param name="pkg_path" value="$(find upm)"/>
        <!-- laser shadow filtering file -->
        <rosparam command="load" file="$(find upm)/yaml/laser_filter.yaml"/>

  	<node pkg="tf" type="static_transform_publisher" name="world_to_laser"
    	args="0.055360992904849 -0.0311190557844302 -0.0198410310814696 
    	-3.12632591028048 -0.00464314210625259 -0.0196141309638589 /camera /laser 10">
	</node>

	<node pkg="upm" type="annotateData" name="annotator" output="screen" args="$(find upm)/bag/UTMFF__2013-01-17-19-00-24.bag" >
	</node>
</launch>
